coalib/processes/Processing.py

```diff
@@ -337,9 +342,10 @@ def instantiate_processes(section,
                           global_bear_list,
                           job_count,
                           cache,
-                          log_printer,
+                          use_raw_files,
```

Comment:
Never add parameters in the middle.  That is a breaking change, which causes errors like the following.  Move it to the end, and add a default value.

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/coalib/coala_main.py", line 122, in run_coala
    debug=debug or args and args.debug)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/coalib/processes/Processing.py", line 763, in execute_section
    debug=debug)
TypeError: instantiate_processes() got multiple values for argument 'console_printer'
```